from moncash.exceptions.moncash_error import MoncashError 

class UnexpectedError(MoncashError):
    pass